LICENSE.txt

```diff
@@ -2223,3 +2223,11 @@ exception of some code pulled in from other repositories (such as
 public domain, released using the CC0 1.0 Universal dedication (*).
 
 (*) https://creativecommons.org/publicdomain/zero/1.0/legalcode
+
+--------------------------------------------------------------------------------
+
+The files in cpp/src/arrow/vendored/fast_float/ contain code from
+
+https://github.com/lemire/fast_float
+
+which is made available under the Apache License 2.0.
```

Comment:
@lemire Is the above blurb acceptable to you, or should we mention a more detailed attribution? Don't hesitate to suggest a wording.

Reply:
@pitrou Yes, yes. This is 100% fine.

Reply:
Thank you!

Reply:
@pitrou For future reference, if I post code publicly on GitHub, you can grab it. I am never going to make a fuss.